### ISSUE TITLE
 few files lost after fine-tuning 

### ISSUE THREAD

**Issue by huacilang** [fengredrum/finetune-whisper-lora#3]
Hello, the directory after fine-tuning only have few files ：adapter_config.json  adapter_model.safetensors  optimizer.pt  preprocessor_config.json  README.md  rng_state.pth  scheduler.pt  trainer_state.json  training_args.bin。
Missing a few files, such as pytorch_model.bin and config.json. What transformations have been applied? 
thanks！
